forms/src/main/java/org/devgateway/toolkit/forms/wicket/page/BasePage.java

```diff
@@ -141,7 +141,7 @@ protected Navbar newNavbar(final String markupId) {
 		NavbarButton<LogoutPage> logoutMenu = new NavbarButton<LogoutPage>(LogoutPage.class,
 				new StringResourceModel("navbar.logout", this, null));
 		logoutMenu.setIconType(GlyphIconType.logout);
-		MetaDataRoleAuthorizationStrategy.authorize(logoutMenu, Component.RENDER, SecurityConstants.Roles.ROLE_EDITOR);
```

Comment:
ROLE_USER was already present so no renaming is necessary.
I did add a script that adds ROLE_USER to every existing user that still don't have it.